perceval/components/_pauli.py

```diff
@@ -31,131 +31,108 @@
 from enum import Enum
 from .linear_circuit import Circuit
 from .unitary_components import BS, PS, PERM
-from .processor import Processor
 from perceval.utils import Matrix
-from typing import List
 
 
 class PauliType(Enum):
+    """
+    Enumeration of different Pauli Types. Usage -
+    1. Choose a Pauli gate (/operator)
```

Comment:
I can't see how your "Usage + 1. and 2. items" correspond to the enum documentation. The enum is just a set of named values, the doc (if needed), should explain what the I, X, Y, Z name stand for.

I'm not asking to document this enum, just to move these few lines of doc to a more appropriate location.